subsys/bluetooth/controller/ll_sw/lll.h

```diff
@@ -163,13 +169,16 @@ struct ull_hdr {
 
 struct lll_hdr {
 	void *parent;
+	uint8_t result;
```

Comment:
Any reason `result` is not conditionally compiled?

Reply:
It would require a lot of additional conditional compiles for use of result, and passing of result in functions. We could do it to save that byte x n and a few CPU cycles, but the code would become less readable. Alternatively, we can add conditional compile to the ull_hdr, and keep it in functions - to save the memory. What do you prefer?